x/group/keeper/msg_server.go

```diff
@@ -471,6 +471,10 @@ func (k Keeper) SubmitProposal(goCtx context.Context, req *group.MsgSubmitPropos
 		return nil, err
 	}
 
+	if err := k.assertMetadataLength(req.Summary, "proposal summary"); err != nil {
```

Comment:
We should add this in the docs as well.

Reply:
two things left to be done like in the gov pr:
 - https://github.com/cosmos/cosmos-sdk/pull/14390/files#diff-9eb4fa821935581a71c53b74e4bef727b55f759276a9912fa47e2159f559f05bR144-R148
 - https://github.com/cosmos/cosmos-sdk/pull/14390/files#diff-3b932fd329b6630207a4705ca1acf774004af9f6e52b7ae7eba0cba8cbb893c1R1158-R1161